src/test/java/org/lsc/beans/BeanComparatorTestOld.java

```diff
@@ -0,0 +1,558 @@
+/*
```

Comment:
What's the point of having the file BeanComparatorTestOld.java? Is this a backup of BeanComparatorTest.java?

If so, we should remove it